Tribler/community/gigachannel/community.py

```diff
@@ -27,18 +34,30 @@ class GigaChannelCommunity(Community):
     Community to gossip around gigachannels.
     """
 
-    master_peer = Peer(unhexlify("3081a7301006072a8648ce3d020106052b8104002703819200040448a078b597b62d3761a061872cd86"
-                                 "10f58cb513f1dc21e66dd59f1e01d582f633b182d9ca6e5859a9a34e61eb77b768e5e9202f642fd50c6"
-                                 "0b89d8d8b0bdc355cdf8caac262f6707c80da00b1bcbe7bf91ed5015e5163a76a2b2e630afac96925f5"
-                                 "daa8556605043c6da4db7d26113cba9f9cbe63fddf74625117598317e05cb5b8cbd606d0911683570ad"
-                                 "bb921c91"))
+    master_peer = Peer(unhexlify("3081a7301006072a8648ce3d020106052b8104002703819200040514b5f2a1d2293976a5627ce9f0148"
```

Comment:
Please use the short key format. You can generate it with the key gen tool in IPv8.

Reply:
Seconded, you can use: `python generate_key.py --public`

Reply:
done